src/routes/graph/update-pin.ts

```diff
@@ -24,6 +25,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
   }).required(),
   body: Joi.object().keys({
     fields: fieldsToServerUpdateSchema.example({ id: 'My port' }),
+    edgeHashId: Joi.string().allow(null).default(null).example('ka08d'),
```

Comment:
This should not have a default I think?

Reply:
So in code It's typed `string | null`. Can be changed to `string | undefined` if you want to drop `default` from Joi.

Should I make such change for `edgeHashId` ?